code/ship/ship.cpp

```diff
@@ -7709,8 +7719,12 @@ void ship_init_cockpit_displays(ship *shipp)
 	ship_set_hud_cockpit_targets();
 }
 
-void ship_clear_cockpit_displays()
+void ship_clear_cockpit_displays(ship* shipp)
```

Comment:
I'm thinking this should be called `ship_close_cockpit_displays`, but maybe that is out of scope for this PR.